e2e/README.md

```diff
@@ -1,7 +1,7 @@
 # Overview
-This crate runs all the end-to-end tests for shuttle. These tests must run against a local dev environment, so you first have to set that up by following [these instructions](../README.md#working-on-shuttle).
+This crate runs all the end-to-end tests for shuttle. These tests must run against a local dev environment, so you first have to set that up by following [these instructions](https://github.com/shuttle-hq/shuttle/blob/main/CONTRIBUTING.md).
```

Comment:
Any reason for linking to `main`? I'm a bit worried about having `rc`s and stuff in the future that might have different instructions for a temporary period.

Reply:
I copied it straight from the base README.  I see what you mean, so I think perhaps we should change both of them to local?

Reply:
Yes, if the change makes sense, then let's change both :+1:

Reply:
Fixed!